nvOC_miner_update.sh

```diff
@@ -809,8 +809,8 @@ function compile-TPccminer {
       echo "Set tpruvot ccminer repo head to $TPccminer_src_hash_ver_8"
       cd ${NVOC_MINERS}/TPccminer/src/
       git reset --hard $TPccminer_src_hash_ver_8
-      git fetch origin $TPccminer_src_hash_ver_8
-      git pull
+      git checkout $TPccminer_src_hash_ver_8    # is this needed?
+      git pull origin $TPccminer_src_hash_ver_8
```

Comment:
You don't need the reset command, and you don't need the pull command, You should only fetch correct branche histories, from future commits, past commits or adjacent branches, and try checking out your target commit (with --force) after every attempt. That's what get-sources function does.